packages/testing/src/execution_testing/execution/transaction_post.py

```diff
@@ -80,6 +85,22 @@ def execute(
         """Execute the format."""
         del fork
         del engine_rpc
+
+        # Check if CLI override for max_transactions_per_batch is provided
+        cli_max_tx_per_batch = request.config.getoption(
+            "max_tx_per_batch", None
+        )
+        if cli_max_tx_per_batch is not None:
+            self.max_transactions_per_batch = cli_max_tx_per_batch
```

Comment:
I'd rather have `max_tx_per_batch` be a property of `BaseExecute`, because all other execute subcommands will use it too, and then we pass it as part of the kwargs here: https://github.com/ethereum/execution-specs/blob/dca59f62f16089e6a3256979b9d218976570140f/packages/testing/src/execution_testing/cli/pytest_commands/plugins/execute/execute.py#L617